releases/firefox-release-rc-49.0.json

```diff
@@ -6,12 +6,11 @@
             "graphid": "",
             "graphid_2": "",
             "human_tasks": {
-                "post_released": false,
-                "published_balrog": false,
+                "published_release": false,
                 "pushed_mirrors": false,
                 "submitted_shipit": false,
                 "emailed_cdntest": false,
-                "published_beta_balrog": false
+                "published_rc_beta_release": false
```

Comment:
this is more confusing to me (rc_beta_release, well which is it?) perhaps "published_rc_to_beta" would be clearer?